packages/nox/src-tpetra/NOX_Tpetra_Vector.hpp

```diff
@@ -287,7 +294,7 @@ class Vector : public Abstract::Vector,
 
 private:
     //! Shared pointer holding the @ref ::Tpetra::Vector wrapped in this object.
-    Teuchos::RCP<vector_type> tpetraVec;
+    Teuchos::RCP<vector_type> tpetraVec = Teuchos::null;
```

Comment:
The default ctor for RCP always initializes to NULL. I don't think this is needed.